### ISSUE TITLE
Does not support babel-plugin-component

### ISSUE THREAD

**Issue by smileShirmy** [element-plus/element-plus#437]
```
  plugins: [
    [
      'component',
      {
        libraryName: 'element-plus',
        styleLibraryName: 'theme-chalk'
      }
    ]
  ]
```

```typescript
import { Button } from 'element-plus';

Vue.use(Button);
```

```shell
This dependency was not found:

* element-plus/lib/button in ./src/main.ts

To install it, you can run: npm install --save element-plus/lib/button
```

**Comment by zazzaz** [comment]
Thanks. We have fixed this issue in `1.0.1-alpha.8`

**Comment by zouhangwithsweet** [comment]
> Thanks. We have fixed this issue in `1.0.1-alpha.8`

for this version, you can use
```
import { ElButton } from 'element-plus';

Vue.use(ElButton);
```
we will fix it in the future
